ivy/utils/exceptions.py

```diff
@@ -349,58 +357,20 @@ def _handle_exceptions(*args, **kwargs):
         """
         try:
             return fn(*args, **kwargs)
-        # Not to rethrow as IvyBackendException
-        except IvyNotImplementedException as e:
-            _configure_stack_trace(e.__traceback__)
-            raise e
-        except IvyError as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except IvyBroadcastShapeError as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyBroadcastShapeError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except IvyDtypePromotionError as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyDtypePromotionError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except (IndexError, IvyIndexError) as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyIndexError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except (AttributeError, IvyAttributeError) as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyAttributeError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except (ValueError, IvyValueError) as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyValueError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except IvyDeviceError as e:
-            _configure_stack_trace(e.__traceback__)
-            raise ivy.utils.exceptions.IvyDeviceError(
-                fn.__name__, str(e), include_backend=True
-            )
-        except InvalidBackendException as e:
+        except (IvyNotImplementedException, InvalidBackendException) as e:
```

Comment:
Not sure why we have a separate except block for these 2 exceptions. I think we can just add `IvyNotImplementedException` to the `_non_ivy_exception_mapping` and handle `InvalidBackendException` through the `IvyException` except block

Reply:
okay, I'll do that